opcua/server/history.py

```diff
@@ -4,6 +4,8 @@
 from opcua import Subscription
 from opcua import ua
 
+import sqlite3
+
```

Comment:
How should I do the import if want to use SQL? I can't add the import statement at the top because then HistoryStorageInterface isn't defined with my SQL class is imported?